lncfg/bitcoind.go

```diff
@@ -4,6 +4,8 @@ package lncfg
 // bitcoind.
 type Bitcoind struct {
 	Dir                string `long:"dir" description:"The base directory that contains the node's data, logs, configuration file, etc."`
+	ConfigPath         string `long:"config" description:"Configuration filepath. If not set, will default to the default filename under "dir"."`
```

Comment:
@dongcarl see this warning, it's a bit tricky, but the mixed quoting is messing things up here.

Reply:
I think you need to use single quotes since all other quote types have a special meaning in this context.